checkov/terraform/checks/resource/aws/KMSRotation.py

```diff
@@ -16,7 +16,7 @@ def get_inspected_key(self):
     def scan_resource_conf(self, conf):
         # Only symmetric keys support auto rotation. The attribute is optional and defaults to symmetric.
         spec = conf.get('customer_master_key_spec')
-        if not spec or 'SYMMETRIC_DEFAULT' in spec:
+        if not spec or 'SYMMETRIC_DEFAULT' in spec or 'HMAC' in spec:
             return super().scan_resource_conf(conf)
         else:
             return CheckResult.PASSED
```

Comment:
also would be great, if you can change the the result to `UNKNOWN`

Reply:
Heya @gruebel,
Re-read the document and agree.  This check isn't relevant for assymetric keys, so once they're identified as not symmetric, the check should not apply.  Is `UNKNOWN` the right result in that instance?

Reply:
Done the switch